### ISSUE TITLE
Document which (inner) blocks are unlocked/removable and why

### ISSUE THREAD

**Issue by nielslange** [woothemes/woocommerce#42632]
See p1653060081672889/1653057704.803339-slack-C8X6Q7XQU

## Notes

Looking at the Cart and Checkout blocks, some of the inner blocks can be moved and/or removed. Currently, it's not documented which (inner) blocks are unlocked/removable and why. This issue aims to document this for future references.

**Comment by zdenys** [comment]
Came here to report a case discussed on Slack p1678436354664229-slack-C3NCP7ZJ6

TL;DR: The Mini Cart Contents block within Mini Cart Template Part of an FSE theme is locked and cannot be unlocked. 


https://user-images.githubusercontent.com/39308239/224475923-9aa55969-ad0c-44f1-a2eb-f122f1090316.mp4



If this is intentional, it should be documented so HEs can point users to public doc where it would be explained. 
- Maybe a mention in https://woocommerce.com/document/woocommerce-blocks/#section-7? 
- Otherwise, it can be mentioned somewhere within the Editor, maybe a tooltip?

**Comment by nielslange** [comment]
> Came here to report a case discussed on Slack p1678436354664229-slack-C3NCP7ZJ6
> 
> TL;DR: The Mini Cart Contents block within Mini Cart Template Part of an FSE theme is locked and cannot be unlocked.
> 
> If this is intentional, it should be documented so HEs can point users to public doc where it would be explained.
> 
> * Maybe a mention in https://woocommerce.com/document/woocommerce-blocks/#section-7?
> * Otherwise, it can be mentioned somewhere within the Editor, maybe a tooltip?

@dinhtungdu, given that you implemented the Mini Cart block, could you answer the previous questions?

**Comment by dinhtungdu** [comment]
@nielslange @zdenys the locking was introduced to fix https://github.com/woocommerce/woocommerce-blocks/issues/6309.

>	If this is intentional, it should be documented so HEs can point users to public doc where it would be explained.

Yes, it's intentional, as the title, products table and footer are required elements of the mini cart content and shouldn't be deleted. While I'm 💯 for updating the documentation, I also wonder why the merchant may need to unlock them. The OP wants to:
- change the mini cart button color: which can be done by setting the color for the Mini Cart Content block because the inner blocks inherit style from the parent.
<img width="1531" alt="image" src="https://user-images.githubusercontent.com/5423135/224630296-a3912efb-febd-4f89-8eb9-d1958a85105a.png">
- adjust the heading: it's not clear what aspect OP wants to adjust, but I think he wants to change its size. If that's the case, it's better to add the font size support for the Mini Cart Title. cc @Aljullu as you also worked on the Mini Cart.

**Comment by Aljullu** [comment]
Thanks for the ping! As @dinhtungdu, it's intentional that those blocks can't be unlocked, as they are part of the critical flow and removing them might cause unexpected consequences and prevent shoppers from completing the order.

Based on the conversation, above:

1. We should document which blocks are locked (and can't be unlocked) and why (this issue).
2. We should allow the user to customize the Mini Cart heading styles and text. We do have a tracking issue for this: https://github.com/woocommerce/woocommerce-blocks/issues/6259.
3. We should allow the user to customize the Mini Cart button styles. I opened a tracking issue for this: https://github.com/woocommerce/woocommerce-blocks/issues/8736.

Please let me know if that doesn't make sense. :slightly_smiling_face:
